learning_loop_node/detector/rest/model_version_control.py

```diff
@@ -20,9 +24,20 @@ class VersionMode(str, Enum):
     Pause = 'pause'  # will pause the updates
 
 
+@dataclass(**KWONLY_SLOTS)
+class ModelVersionResponse:
+    current_version: str = field(metadata={"description": "The current version of the model."})
+    target_version: str = field(metadata={"description": "The target version of the model."})
+    loop_version: str = field(metadata={"description": "The version of the loop."})
+    local_versions: List[str] = field(metadata={"description": "The local versions of the model."})
+    version_control: str = field(metadata={"description": "The version control mode."})
```

Comment:
Probably fine since the differentiation between loop and model IS important here.
Still, for when we talk about it: `Adding "The {variable name} of the mode / loopl" does not add much inforamtion`.